.prettierignore

```diff
@@ -0,0 +1 @@
+*.vue
```

Comment:
Any special reason we added this? (sorry for being late 😅)

Reply:
They aren't formatted currently, and prettier doesn't work properly with vue's template part.